Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptAddView.jsp

```diff
@@ -8,20 +8,43 @@
 
 <script>
 	$(function() {
+		$(document)
+				.ready(
+						function() {
+							var term = $('#term').val();
+							if (term != '') {
+								console.log('term');
+								console.log(term);
+								var synonymDescription = $(
+										'#synonymDescription').val();
+								var id = $('#synonymId').val();
+								var termArray = term.split(',,');
+								console.log('term array');
+								console.log(termArray);
+								var synonymDescriptionArray = synonymDescription
+										.split(',,');
+								var idArray = id.split(',,');
+								console.log(termArray.length);
+								for (var i = 0; i < termArray.length; i++) {
+									if (idArray[i] != '') {
+										synonymAdd(idArray[i], termArray[i],
+												synonymDescriptionArray[i],
+												'true');
+									}
+								}
+							}
+						});
```

Comment:
please fix the indentations. 4 spaces and for some reason the Eclipse formatting puts too much white space when starting functions etc. this should be readable.